node_lambnik/scripts/test

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+function usage() {
+    echo -n "Usage: $(basename "${0}") [TEST]
+Run specified test.
+Options:
+    --help      Display this help text
+"
+}
+
+function main() {
+	mkdir -p test-results
+
+    if [ "${1}" = "local-tile" ]
+    then
+        curl http://localhost:9001/tile/13/2385/3102.png "Accept: image/png" > test-results/local-tile.png
+    elif [ "${1}" = "prod-tile" ]
+    then
+        curl https://adzvdcdhi4.execute-api.us-east-1.amazonaws.com/latest/tile/13/2385/3102.png -H "Accept: image/png" > test-results/prod-tile.png
+    else
+    	usage
+    fi
+
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        SERVER_COMMAND=${1:-start}
+        main $SERVER_COMMAND
+    fi
+    exit
+fi
```

Comment:
I think it might be better to leave this out of this PR and save it for issue #14.  Whether it stays or goes, I'd rather that it clean up after itself than create a `test-results` directory that sticks around.

Reply:
I'll remove it, all the scripts do are try to download the images from the tile server and put them in that directory, it's sort of vestigial from before the binary content situation was sorted out.